### ISSUE TITLE
Project recreates scriptable object when the project is pulled from git

### ISSUE THREAD

**Issue by oktayturkdagli** [mob-sakai/SoftMaskForUGUI#200]
**Describe the bug**
When a new project is created and sent to git and then the project is pulled by someone else, the asset in the project settings folder appears as a white file in Unity and Unity does not recognize this file, the soft mask creates a new asset. So we have two assets but only one of them works.

**To Reproduce**
I have not tried to reproduce it, I can provide more information about it later. But I think it should be like this:
Both sides have pulled the project once. Then one of the users changes the motion sensitivity to High on the asset and pushes it. The person on the other side pulls and sees that their asset is damaged and a new asset is created.

**Environment:**
Version: 2.3.7
Platform: Mac
Unity version: Unity 2022.3.45
Build options: [ IL2CPP, .Net 4.x, URP, Android(API 33)]

**Comment by mob-sakai** [comment]
Thank you for your reporting!
Indeed, an exception occurred immediately after the package installation.

**Comment by mob-sakai** [comment]
@oktayturkdagli 
Please try RC version.
```
"com.coffee.softmask-for-ugui": "https://github.com/mob-sakai/SoftMaskForUGUI.git?path=Packages/src#develop",
```

**Comment by oktayturkdagli** [comment]
Great! I'll try on Monday. 🙏

**Comment by oktayturkdagli** [comment]
There doesn't seem to be a problem here, but I haven't tested it in detail. I can do detailed tests after the release.

**Comment by Mepkatatsu** [comment]
I experienced the same issue in version 2.3.8. I'm not sure of the exact cause(I had moved the package to a different path when I first import it.), it seemed that every time I pulled from Git or restarted the project, the UISoftMask.asset was always newly generated, but it wasn't properly registered to the Preloaded Assets. So I manually registered it to Preloaded Assets, and the issue was resolved.
